### PR TITLE
[Milestone 4]: Join tables

### PR DESCRIPTION
**Pull request by RichardChileya**
### In this section, I : 
- Created join tables for specialization, vets and visits.
- I Updated the queries to check for data in join tables.
- I inserted data into the created tables. 
- I updated the animals'
- [ ] create visits table
create visits table
- [ ] create a specialization table
create specializations table
- [ ] create a vets table
create vet table

- [ ] specialities 
- maisy smith specialties
- [ ] animals seen by william
animal seen by william
- [ ] visits data 
visits data
- [ ] not specialized 
not specialized
- [ ] most recent visits
most recent visit
- [ ] most visits
most visits
- [ ] list all visits
List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020
- [ ] animals seen by tatcher
animal seen by tatcher
- [ ] vets and specialties
vets and specialties
- [ ] animals seen by stephen
animals seen by stephen

- [ ] Maisy smith first visit
Screenshot (200)

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Richard, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please add screenshots of queries results to the pull request description.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there @RichardChileya 👋🏽 , 

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- You have added some screenshots
- You created queries
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [x] Kindly ensure that you add screenshots of ALL the queries run to your PR description per the requirements.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 1]
```diff
@@ -156,4 +156,99 @@ SELECT owners.full_name , COUNT(*)
...
+
+  -- [X] Who was Maisy Smith's first visit?
+SELECT vets.name AS "Vet", animals.name AS "Animal", visits.date_of_visit
+  FROM vets
+  JOIN visits
+    ON vets.id = visits.vet_id
+  JOIN animals
+    ON visits.animal_id = animals.id
+  WHERE vets.name = 'Maisy Smith'
+  ORDER BY date_of_visit LIMIT 1;
```
Comment (1GORDON) [inline queries.sql:216]: Great job @RichardChileya 🤝

- [x] [Optional] You may kindly add a screenshot of Who was Maisy Smith's first visit?

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @RichardChileya ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

- [x] You may also fix the broken MIT license link to make your readme file more professional.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**